infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -146,19 +146,27 @@
      */
     public SequencerServer(ServerContext serverContext) {
         this.serverContext = serverContext;
-        Config config = Config.parse(serverContext.getServerConfig());
 
         // Sequencer server is single threaded by current design
         this.executor = Executors.newSingleThreadExecutor(
                 new ServerThreadFactory("sequencer-", new ServerThreadFactory.ExceptionHandler()));
 
+        globalLogTail = 0;
 
-        globalLogTail = config.getInitialToken();
+        this.cache = new SequencerServerCache(serverContext.getConfiguration().getSequencerConflictWindowSize());
 
-        this.cache = new SequencerServerCache(config.getCacheSize());
+        setUpTimerNameCache();
+    }
 
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4662121)